package.json

```diff
@@ -13,7 +13,8 @@
   "dependencies": {
     "react": "*",
     "react-dom": "*",
-    "react-router": "*"
+    "react-router": "*",
+    "webpack-dev-server": "^1.14.0"
```

Comment:
This is a duplicate of what was in the dev dependencies. Was this specific version required given that problem that you were having with the "replace" error earlier tonight? This should still be in the dev deps either way